RoddyCore/src/com/stackoverflow/questions/xmlvalidation/LSInputImpl.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 eilslabs.
+ * Copyright (c) 2015 German Cancer Research Center (Deutsches Krebsforschungszentrum, DKFZ).
  *
  * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
```

Comment:
Wrong paths to Github repo (everywhere!)

https://www.github.com/TheRoddyWMS/Roddy/LICENSE.txt